### ISSUE TITLE
run bot on a new server

### ISSUE THREAD

**Issue by NWuensche** [derhuerst/vbb-telegram#7]
When I connect in Telegram to the public_transport_bot and `/start`, nothings happens. Also stuff like `/route` doesn't do anything.

**Comment by derhuerst** [comment]
Yes, unfortunately. The bot is currently running on a server that has [an IP address blocked by VBB](
https://github.com/derhuerst/vbb-rest/issues/29#issuecomment-406795226).
